Gemfile

```diff
@@ -4,6 +4,7 @@ gemspec
 
 group :development, :test do
   gem 'appraisal', '~> 2.1'
+  gem 'byebug'
```

Comment:
I think this should be omitted. It feels like the tool(s) used for debugging comes down to the preference of the individual.

Reply:
Fine to remove it, just convenient to have when locally tracing code paths, and running the tests locally with `bundle exec` means it won't be available if it's not in the `Gemfile`.  Stuff in the `Gemfile` doesn't get pulled into the gem release either, only relevant for someone developing locally.